### PR TITLE
Updated inactive anchor links and changed old url paths in admin-onboarding-tasks.rst

### PR DESCRIPTION
**Pull request by krutarththakkar**
Updated Old Url Paths with Anchors for better navigation

### CODE REVIEW

**Comment by krutarththakkar** [comment]
The old links on the site redirect to new paths, hence the anchor tags dont work out.
Hence, I updated the links to the new paths, and added relevant anchors so the user can navigate to the desired information like before, on clicking the links.

File: source/getting-started/admin-onboarding-tasks.rst
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:90]:
```suggestion
  - Enable audio/video calling and screen sharing with `Jitsi <https://mattermost.com/blog/mattermost-and-jitsi/>`__ or `Zoom <https://mattermost.com/marketplace/zoom-plugin/>`__.
```

**Review by justinegeffen — COMMENTED** [review]
Just one suggestion otherwise this looks good to me, thank you! :)

**Comment by krutarththakkar** [comment]
Thanks, committed your suggestion and one new commit,
Added missing anchors where needed.
Please review final changes :)

File: source/getting-started/admin-onboarding-tasks.rst
Comment (justinegeffen) [inline source/getting-started/admin-onboarding-tasks.rst:91]: Curious as to why the following three links weren't changed along with the previous two?

File: source/about/corporate-directory-integration.rst
Comment (justinegeffen) [inline source/about/corporate-directory-integration.rst:22]: This doesn't seem like a valid URL. Please double-check and provide the correct one. Thank you! :)

**Comment by krutarththakkar** [comment]
How about now?
I guess most things are resolved!

File: source/getting-started/admin-onboarding-tasks.rst
Comment (justinegeffen) [reply source/getting-started/admin-onboarding-tasks.rst:91]: @krutarththakkar, would you be open to updating all the links here so they have the correct URLs (i.e., do not include the search string but are the direct URL). Thank you. :)

**Comment by justinegeffen** [comment]
Thanks, @krutarththakkar. I have provided a second review and comment about the four Marketplace links. Let me know if you have any questions! :)

File: source/getting-started/admin-onboarding-tasks.rst
Comment (krutarththakkar) [reply source/getting-started/admin-onboarding-tasks.rst:91]: Ok, Am on it!

**Comment by krutarththakkar** [comment]
Updated everything,
Awaiting final review.

**Review by justinegeffen — APPROVED** [review]
This is a great PR! Thank you for all the effort you put into it. :)

**Pull request merged**
